README.md

```diff
@@ -28,9 +28,7 @@ under the License.
 [![Get on Slack](https://img.shields.io/badge/slack-join-orange.svg)](http://bit.ly/join-superset-slack)
 [![Documentation](https://img.shields.io/badge/docs-apache.org-blue.svg)](https://superset.apache.org)
 
-<picture
-  width="500"
->
+<picture width="500">
```

Comment:
@andy-clapson, @bipinct recently made this change in https://github.com/apache/superset/pull/25721 to address a rendering error.

Reply:
Well now it's super DUPER fixed